src/overlays/actors/ovl_En_Dai/z_en_dai.h

```diff
@@ -5,9 +5,43 @@
 
 struct EnDai;
 
+typedef void (*EnDaiActionFunc)(struct EnDai* this, GlobalContext* globalCtx);
+
+typedef struct EnDaiParticle {
+    /* 0x00 */ u8 enabled;
```

Comment:
```suggestion
    /* 0x00 */ u8 isEnabled;
```
Similar to a comment left by @engineer124 in your other review, just to maintain consistency